copter/source/docs/parachute.rst

```diff
@@ -54,8 +54,10 @@ set the following parameters.
 -  :ref:`CHUTE_TYPE <CHUTE_TYPE>` = "10" to release with a servo
 -  :ref:`CHUTE_SERVO_ON <CHUTE_SERVO_ON>` should be set to the servo position required to release the parachute
 -  :ref:`CHUTE_SERVO_OFF <CHUTE_SERVO_OFF>` should be the "resting" servo position.  I.e. the position the servo is in before the parachute is released
--  if using AC3.5 (or higher) set :ref:`SERVO9_FUNCTION <SERVO9_FUNCTION>` = "27".  `SERVO9_` refers to the Pixhawk's AUX OUT 1.  To use AUX OUT2 instead set :ref:`SERVO10_FUNCTION <SERVO10_FUNCTION>` to 27, etc.
--  if using AC3.4.6 (or lower) set RC9_FUNCTION = "27".  `RC9_` refers to the Pixhawk's AUX OUT 1.  To use AUX OUT2 instead set RC10_FUNCTION to 27, etc.
+-  set :ref:`SERVO9_FUNCTION <SERVO9_FUNCTION>` = "27".  `SERVO9_` refers to the Pixhawk's AUX OUT 1.  To use AUX OUT2 instead set :ref:`SERVO10_FUNCTION <SERVO10_FUNCTION>` to 27, etc.
+-  :ref:`CHUTE_CRT_SINK<CHUTE_CRT_SINK>` should be set to the maximum sink rate in m/s that will trigger the chute release.
```

Comment:
I wonder if it's a "maximum" or a "minimum"?  I think maybe we should just remove the word "maximum"

Reply:
done